src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -592,7 +593,7 @@ private static XmlSerializer[] GetReflectionBasedSerializers(XmlMapping[] mappin
             return serializers;
         }
 
-        internal static bool GenerateSerializer(Type[] types, XmlMapping[] mappings, Stream stream)
+        internal static bool GenerateSerializer(Type[]? types, XmlMapping[] mappings, Stream stream)
```

Comment:
Why is `types` nullable here? We checked that there are no references to this method anywhere though, so it's not being used in an `if` block anywhere too. I think `types` can be non-nullable here.

Reply:
if the method can handle `types == null` there is no harm in annotating it as nullable. It looks like dead code to me anyway but will not remove it in this PR as there is lots of reflection happening here so it might actually be used in some cryptic way